runtime/vm/CMakeLists.txt

```diff
@@ -126,6 +126,16 @@ add_library(j9vm SHARED
 )
 
 
+#JIT helper methods require us to disable buffer security checks on Windows
```

Comment:
```suggestion
# JIT helper methods require us to disable buffer security checks on Windows
# See https://github.com/eclipse/openj9/pull/1494 for rationale
```

Reply:
Suggest adding the link to the original PR as we lose the blame path in the cmake rewrite

Reply:
fixed